modules/ROOT/pages/async-scope-reference.adoc

```diff
@@ -4,48 +4,49 @@ include::_attributes.adoc[]
 
-== Async Configuration
+== Reference
 
 Async scopes are configurable.
 
-[%header,cols="1,4"]
+[%header,cols="1a,1a,1a,4a"]
 |===
-| Field | Description
-| Display Name (`name`) | Name for the Async scope.
-| Max Concurrency (`maxConcurrency`) a| Optional. Sets the maximum number of concurrent messages that the scope can process. By default, the container thread pool determines the maximum number of threads to use to optimize the performance when processing messages. When the scope is processing the maximum number of concurrent messages, it cannot receive additional requests.
+| Field | XML | Default | Description
+| *Display Name* | `name` | Async | Name for the Async scope.
+| *Max Concurrency* | `maxConcurrency` | See description. |  Optional. Sets the maximum number of concurrent Mule messages or variables that the scope can process. By default, the container thread pool optimizes performance by determinng the maximum number of threads to use. 
 
-Set `maxConcurrency` to `1` to cause the scope to process requests one at a time.
+Setting `maxConcurrency` to `1` causes the scope to process requests one at a time.
 
-See xref:execution-engine.adoc#backpressure[Back-Pressure Management] for details about Mule behavior after reaching the maximum concurrency value.
+Upon reaching maximum concurrency,the scope cannot receive additional requests. See xref:execution-engine.adoc#backpressure[Back-Pressure Management] for details about the behavior of Mule when it reaches the maximum concurrency value.
 |===
 
 == Async Scopes versus Subflows
 
-Unlike a subflow, an Async scope:
+Unlike the xref:flow-component.adoc[Subflow}, the Async scope:
 
 * Does not inherit the exception strategy of the main flow.
 +
-To handle errors in an Async scope, use the Try scope.
+To handle errors in an Async scope, use the xref:try-scope-concept.adoc[Try scope].
 +
 * Processes messages asynchronously.
 * Does not pass data back to the main flow.
 * Exists inline with the main flow thread.
-* Is not called by a Flow Reference component.
-* Is not reusable
+* Is not called by a xref:flowref-about.adoc[Flow Reference] component.
+* Is not reusable.
 
-Note that even though the Async scope receives a copy of the Mule message, the payload is not copied. The same payload objects are referenced by both Mule messages: One that continues down the original flow, and the one processed by the Async scope.
 
-In other words, if the payload of your message is a mutable object (for example, a bean with different fields in it) and a message processor in your Async scope changes the value of one of the fields, the message processors outside of the Async scope see the changed values.
+// TODO: DOES THIS INFO BELONG IN THIS SECTION?
+// TODO: Do we have a recommendation here?
```

Comment:
TODO: @IvanAndresFritzler we can say something like copy the payload if going to mutate it, only applies if doing something after the async. would need to add custom code for this.